lib/OpenQA/Script/CloneJob.pm

```diff
@@ -226,11 +226,32 @@ sub handle_tx ($tx, $url_handler, $options, $jobs) {
     }
 }
 
+# append a formatted "-NN" to the TEST parameter
+sub append_sequential_number_to_test_name ($n, $post_params) {
+    my $number_suffix = sprintf("-%02d", $n);
+    foreach my $job (keys %$post_params) {
+        my $test_name = $post_params->{$job}->{'TEST'};
```

Comment:
I'd avoid repeated hash access and unnecessary quoting (which tidy will likely complain about as well):
```suggestion
    foreach my $job_key (keys %$post_params) {
        my $job = $post_params->{$job_key};
        my $test_name = $job->{TEST};
```
Then use `$job` in further code instead of repeating `$post_params->{$job}`.